src/main/java/com/taskadapter/redmineapi/bean/Issue.java

```diff
@@ -616,13 +613,15 @@ public PropertyStorage getStorage() {
      * @throws NotFoundException       the required project is not found
      * @throws RedmineException
      */
-    public Issue create() throws RedmineException {
-        return transport.addObject(this, new BasicNameValuePair("include",
-                Include.attachments.toString()));
+    public Issue create(NameValuePair... params) throws RedmineException {
+        NameValuePair[] enrichParams = Arrays.copyOf(params, params.length + 1);
+        enrichParams[params.length] = new BasicNameValuePair("include",
+                Include.attachments.toString());
+        return transport.addObject(this, enrichParams);
```

Comment:
1. 
I am fine with the general idea, but let's remove apachehttp-specific NameValuePair from Issue class params.
I see so far NameValuePair is only used deep inside Transport internals, like URICOnfigurator or Transport class itself. 
it is better to keep "user-facing" classes like `Issue` free of transport details. we may want to replace Apache HttpCore library later.

2. 
need tests to ensure the methods work fine with no params, null param, *some* params, etc.